### PR TITLE
Fixed Animekimi

### PR DESCRIPTION
**Pull request by Faelayis**
- Improve code
- Improve iframe
- Fixed tag page wrong
- Fixed subtitle not show
- Fixed search not show presence

<details>
<summary>
Preview
</summary>
<br>
Fixed tag page wrong

image
Fixed subtitle not show
image
Fixed search not show presence
image
</details>

### CODE REVIEW

File: websites/A/Animekimi/presence.ts [hunk 1]
```diff
@@ -41,54 +26,72 @@ presence.on("UpdateData", async () => {
...
+    playvdo =
+      document.querySelector("#info > h1")?.textContent ?? "ไม่ทราบชื่อเรื่อง",
+    ep =
+      document.querySelector(
+        "#single > div.content > div.sheader > div.data > h1"
+      )?.textContent ?? "ไม่ทราบชื่อตอน",
+    search =
+      document.querySelector(
+        ".content.rigth.csearch > header > h1" ||
+          ".no-result.animation-2 > h2 > span"
```
Comment (Slowlife01) [inline websites/A/Animekimi/presence.ts:46]: I don't think `||` will even work here

File: websites/A/Animekimi/presence.ts [hunk 2]
```diff
@@ -41,54 +26,72 @@ presence.on("UpdateData", async () => {
...
-  } else if (path.pathname.includes("catalog")) {
-    presenceData.details = "หมวดหมู่ ";
+    presenceData.details = "ประเภท";
     presenceData.state = title;
-  } else if (path.pathname.includes("category")) {
-    presenceData.details = "หมวดหมู่ ";
-    presenceData.state = title;
-  } else if (path.pathname.includes("title")) {
-    presenceData.details = "หมวดหมู่ ";
+  } else if (path.pathname.includes("catalog" || "category" || "title")) {
```
Comment (Slowlife01) [inline websites/A/Animekimi/presence.ts:64]: Same here

File: websites/A/Animekimi/presence.ts [hunk 3]
```diff
@@ -41,54 +26,72 @@ presence.on("UpdateData", async () => {
...
-
-      if (moive.includes("ซับไทย"))
-        moive = moive.replace("เดอะมูฟวี่ ซับไทย", "").trim();
-      else if (moive.includes("พากย์ไทย"))
-        moive = moive.replace("เดอะมูฟวี่ พากย์ไทย", "").trim();
-
-      moive = `เดอะมูฟวี่ ${moive}`;
-      presenceData.details = moive;
+    if (titlemovies.includes("มูฟวี่" || "เดอะมูฟวี่")) {
+      const movieinfo = titlemovies.split("มูฟวี่" || "เดอะมูฟวี่");
```
Comment (Slowlife01) [inline websites/A/Animekimi/presence.ts:83]: Same here

Comment (Faelayis) [reply websites/A/Animekimi/presence.ts:84]: I fixed the presence It won't work the way I want.

Comment (Slowlife01) [reply websites/A/Animekimi/presence.ts:84]: No but like that isn't how `||` works

Comment (Faelayis) [reply websites/A/Animekimi/presence.ts:84]: What should I do

File: websites/A/Animekimi/presence.ts [hunk 4]
```diff
@@ -41,54 +26,73 @@ presence.on("UpdateData", async () => {
...
```
Comment (Slowlife01) [inline websites/A/Animekimi/presence.ts:84]:
```suggestion
    if (titlemovies.includes("มูฟวี่")) {
      const movieinfo = titlemovies.split(/(เดอะ)?(มูฟวี่)/);
```

**Review by Slowlife01 — APPROVED** [review]

**Review by Bas950 — APPROVED** [review]

**Pull request merged**